frameworks/kafka/docs/configure.md

```diff
@@ -27,82 +27,53 @@ The Kafka scheduler runs as a Marathon process and can be reconfigured by changi
+1.  Choose a `DEPLOY_STRATEGY`: serial, serial-canary, parallel-canary, or parallel. See the SDK Developer guide for more information on [deployment plan strategies](https://mesosphere.github.io/dcos-commons/developer-guide.html#plans). <!-- I'm not sure I like this solution, since users aren't going to have the context for the dev guide). -->
 1.  Click **REVIEW & RUN** to apply any changes and cleanly reload the Kafka scheduler. The Kafka cluster itself will persist across the change.
 
-## Configuration Deployment Strategy
-
-Configuration updates are rolled out through execution of Update Plans. You can configure the way these plans are executed.
-
-## Configuration Update Plans
-
-In brief, "plans" are composed of "phases," which are in turn composed of "steps." Two possible configuration update strategies specify how the steps are executed. These strategies are specified by setting the `PHASE_STRATEGY` environment variable on the scheduler. By default, the strategy is `INSTALL`, which rolls changes out to one broker at a time with no pauses.
-
-The alternative is the `STAGE` strategy. This strategy injects two mandatory human decision points into the configuration update process. Initially, no configuration update will take place: the service waits for a human to confirm the update plan is correct. You may then decide to either continue the configuration update through a REST API call, or roll back the configuration update by replacing the original configuration through the DC/OS web interface in exactly the same way as a configuration update is specified above.
-
-After specifying that an update should continue, one step representing one broker will be updated and the configuration update will again pause. At this point, you have a second opportunity to roll back or continue. If you decide to continue a second time, the rest of the brokers will be updated one at a time until all the brokers are using the new configuration. You may interrupt an update at any point. After interrupting, you can choose to continue or roll back. Consult the "Configuration Update REST API" for these operations.
-
 ## Configuration Update REST API
 
-There are two phases in the update plans for Kafka: Mesos task reconciliation and update. Mesos task reconciliation is always executed without need for human interaction.
-
-Make the REST request below to view the current plan. See the REST API Authentication part of the REST API Reference section for information on how this request must be authenticated.
+Make the REST request below to view the current deployment plan. See the REST API Authentication part of the REST API Reference section for information on how this request must be authenticated.
 
-    $ curl -H "Authorization: token=$AUTH_TOKEN" "$DCOS_URI/service/kafka/v1/plan"
-    GET $DCOS_URI/service/kafka/v1/plan HTTP/1.1
+    curl -H "Authorization: token=$auth_token" "<dcos_url>/service/kafka/v1/plan"
```

Comment:
/service/kafka/v1/plan is an alias to /service/kafka/v1/plans/deploy 
I would use /service/kafka/v1/plans/deploy here